screens/intro.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import { StyleSheet, Text, View } from "react-native";
+import {
+  createStackNavigator,
+  createSwitchNavigator,
+  createAppContainer
+} from "react-navigation";
+import Splash from "./Splash";
+import Login from "./Login";
+
+class Intro extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentScreen: "splash"
+    };
+    setTimeout(() => {
+      this.setState({
+        currentScreen: "Login"
+      });
+    }, 2000);
+  }
+
+  render() {
+    console.log(this.props.navigation);
+    const { currentScreen } = this.state;
+    let mainscreen = currentScreen === "splash" ? <Splash /> : <Login />;
```

Comment:
this splash screen is not necessary, because Expo provides a configurable splash screen. But if you intend to use it as a loading screen, it's cool - but not with the `setTimeout()` implementation.

Reply:
Noted. I have removed the `intro.js` file since it was unnecessary